frontend/device/generic/device.lua

```diff
@@ -606,6 +606,28 @@ function Device:_setEventHandlers(UIManager)
         UIManager.event_handlers.PowerOff = function() end
     end
 
+    if self:canRestart() then
+        UIManager.event_handlers.Restart = function(message_text)
```

Comment:
This actually creates a handler for a key named `"Restart"`, which is not something we currently have anywhere (I think).

It's not necessarily an issue, and I understand the symmetry with Reboot/PowerOff, so, just mentioning it here ;).